docs/examples/rows/row_dragging_entire_row.py

```diff
@@ -0,0 +1,49 @@
+"""
+Entire Row Dragging
+"""
+
+import dash_ag_grid as dag
+from dash import Dash, dcc, html
+import pandas as pd
+
+app = Dash(__name__)
+
+df = pd.read_csv(
+    "https://raw.githubusercontent.com/plotly/datasets/master/ag-grid/olympic-winners.csv"
+)
+
+columnDefs = [
+    {'field': 'athlete'},
+    {'field': 'country'},
+    {'field': 'year', 'width': 100},
+    {'field': 'date'},
+    {'field': 'sport'},
+    {'field': 'gold'},
+    {'field': 'silver'},
+    {'field': 'bronze'},
+]
+
+defaultColDef = {'width': 170, "sortable": True, "filter": True}
+
+app.layout = html.Div(
+    [
+        dcc.Markdown("This grid shows the row dragging on entire row"),
+        dag.AgGrid(
+            id='grid-row-dragging',
```

Comment:
```suggestion         
```
This id need to be deleted (or changed) since it's the same id as in the row_dragging.py example.  The duplicate id causes that callback to fail.